specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -34,6 +34,42 @@
     }
   },
   "paths": {
+    "{scope}/providers/Microsoft.ManagedIdentity/identities": {
```

Comment:
We don't support this API (list system assigned identities). We only support `GET {scope}/providers/Microsoft.ManagedIdentity/identities/default`